simpleast-core/src/main/java/com/discord/simpleast/code/TypeScript.kt

```diff
@@ -0,0 +1,249 @@
+package com.discord.simpleast.code
+
+import com.discord.simpleast.code.CodeRules.toMatchGroupRule
+import com.discord.simpleast.core.node.Node
+import com.discord.simpleast.core.node.StyleNode
+import com.discord.simpleast.core.parser.ParseSpec
+import com.discord.simpleast.core.parser.Parser
+import com.discord.simpleast.core.parser.Rule
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+object TypeScript {
+
+  val KEYWORDS: Array<String> = arrayOf(
+     "import|from|export|default|package",
+     "class|enum",
+     "function|super|extends|implements|arguments",
+     "var|let|const|static|get|set|new",
+     "return|break|continue|yield|void",
+     "if|else|for|while|do|switch|async|await|case|try|catch|finally|delete|throw|NaN|Infinity",
+     "of|in|instanceof|typeof",
+     "debugger|with",
+     "true|false|null|undefined",
+     "type|as|interface|public|private|protected|module|declare|namespace",
+     "abstract|keyof|readonly|is|asserts|infer|override"
```

Comment:
```suggestion
     "abstract|keyof|readonly|is|asserts|infer|override|intrinsic"
```
Forgot to mention `intrinsic` last review.